app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V1
+    class RegistrationsController < ApplicationController
```

Comment:
```suggestion
    class RegistrationsController < ApiController
```
ApiController:
 class ApiController < ActionController::API